pkg/api/types.go

```diff
@@ -940,10 +940,13 @@ type ContainerStatus struct {
 	Ready bool `json:"ready"`
 	// Note that this is calculated from dead containers.  But those containers are subject to
 	// garbage collection.  This value will get capped at 5 by GC.
-	RestartCount int    `json:"restartCount"`
-	Image        string `json:"image"`
-	ImageID      string `json:"imageID"`
-	ContainerID  string `json:"containerID,omitempty"`
+	RestartCount      int              `json:"restartCount"`
+	Image             string           `json:"image"`
+	ImageID           string           `json:"imageID"`
+	ContainerID       string           `json:"containerID,omitempty"`
+	ImageCreationTime unversioned.Time `json:"imageCreationTime,omitempty"`
```

Comment:
It makes sense but moving the Image ID to the imageInfo struct would break BWC. Would it still make sense if the Image ID was not part of that struct?